### ISSUE TITLE
Mis-classified log files for sem2lidar and cam2lidar in README.md

### ISSUE THREAD

**Issue by liuQuan98** [hancyran/LiDAR-Diffusion#18]
Great work and open-source! I really appreciate it.

However, I found that your produced log files for sem2lidar and cam2lidar are swapped in the README file, _i.e._, the 5.4 Gig log contains sem2lidar (because the original_conditioning.pngs are semantic maps) and the 2.1 Gig log contains cam2lidar, but they are listed as the exact opposite in the [table](https://github.com/hancyran/LiDAR-Diffusion?tab=readme-ov-file#pretrained-lidms-for-other-tasks). Could you please update the README.md to fix it? I also want to try if the model ckpts are swapped as well, but sadly I have no gpus at hand and cannot test it. I would appreciate it if you could be kind enough to check that as well.

**Comment by Haiyunn** [comment]
@liuQuan98 Hello, have you resolved the issue mentioned above? I found that in the cam2lidar log file, there is a mismatch between the images and the sampled point clouds, and they do not correspond one-to-one. I am not sure whether this is due to the author incorrectly labeling the frames or if it is caused by inaccurate cam2lidar sampling.

**Comment by liuQuan98** [comment]
> [@liuQuan98](https://github.com/liuQuan98) Hello, have you resolved the issue mentioned above? I found that in the cam2lidar log file, there is a mismatch between the images and the sampled point clouds, and they do not correspond one-to-one. I am not sure whether this is due to the author incorrectly labeling the frames or if it is caused by inaccurate cam2lidar sampling.

@Haiyunn No, as I no longer use the official logs now. It's easy to train and generate data with our own gpus.

**Comment by Haiyunn** [comment]
@liuQuan98 Thank you for your reply. I’m wondering if it would be convenient for you to send the results of your sampling, as I would like to compare them. Due to time and GPU limitations, I haven’t been able to reproduce this work yet. Thank you very much
